CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 * Update avifCropRectConvertCleanApertureBox() to the revised requirements in
   ISO/IEC 23000-22:2019/Amd. 2:2021 Section 7.3.6.7.
+* Fix potential heap buffer overflows reported in crbug.com/1501766 and
+  crbug.com/1501770.
```

Comment:
This bug is not a heap buffer overflow. Should we replace "heap buffer overflows" with the more generic "memory errors"?